### PR TITLE
Cleanup metadata.json/README.md after migration

### PR DESCRIPTION
**Pull request by bastelfreak**
Fixes https://github.com/voxpupuli/puppet-consul/issues/576

### CODE REVIEW

**Review by solarkennedy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -499,3 +504,12 @@ a module that can do that.
 Open an [issue](https://github.com/solarkennedy/puppet-consul/issues) or
 [fork](https://github.com/solarkennedy/puppet-consul/fork) and open a
 [Pull Request](https://github.com/solarkennedy/puppet-consul/pulls)
+
+
+## Transfer Notice
+
+This plugin was originally authored by [KyleAnderson](http://github.com/KyleAnderson).
```
Comment (alexjfisher) [inline README.md:511]: Plugin?

File: README.md [hunk 2]
```diff
@@ -499,3 +504,12 @@ a module that can do that.
...
+The maintainer preferred that Vox Pupuli take ownership of the module for future improvement and maintenance.
+Existing pull requests and issues were transferred over, please fork and continue to contribute here instead of Camptocamp.
```
Comment (alexjfisher) [inline README.md:513]: @bastelfreak Camptocamp??

File: README.md (hunk 1 above)
Comment (solarkennedy) [reply README.md:511]: I'm totally fine without this notice here. My username is `solarkennedy` anyway, which makes it not even right! :)

Comment (bastelfreak) [reply README.md:511]: was a bit confusing that the forge username is different to the GitHub handle :D I will update it

**Pull request merged**